sqlglot/parser.py

```diff
@@ -327,6 +330,22 @@ def _parse_insert(self):
             overwrite=overwrite,
         )
 
+    def _parse_update(self):
+        self._match(TokenType.UPDATE)
+        this = self._parse_table(None)
+
+        self._match(TokenType.SET)
+        set_expression = self._parse_expression()
```

Comment:
one statement

return exp.Update(
  this=self._parse_table(None),
  expressions=self._parse_csv(self._parse_equality),
  where=self._parse_where(),
)

Reply:
Great idea 😄